pydantic/types.py

```diff
@@ -936,6 +1048,18 @@ def condate(
     lt: date | None = None,
     le: date | None = None,
 ) -> type[date]:
+    """A wrapper for date that adds constraints.
+
+    Args:
+        strict: Whether to validate the value in strict mode. Defaults to `None`.
```

Comment:
To be consistent with the other docstrings
```suggestion
        strict: Whether to validate the date value in strict mode. Defaults to `None`.
```